### PR TITLE
Documentation for query field name and datatype in query shape

### LINKED ISSUE #8539: [DOC] Query field name and type in query insights documentation
**What do you want to do?**
 As a followup to : https://github.com/opensearch-project/documentation-website/pull/8173
Need to add documentation for field name and field type settings and field that is included in the query structure while grouping top b by similarity.

- Need to add documentation for the 2 settings
- Need to modify the example query shape to now show field name and field data type and mention that we take this into account for the query shape.

- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request.

***Version:** List the OpenSearch version to which this issue applies, e.g. 2.14, 2.12--2.14, or all.
 
**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by deshsidd**
### Description
As a followup to : https://github.com/opensearch-project/documentation-website/pull/8173
Need to add documentation for field name and field type settings and field that is included in the query structure while grouping top b by similarity.

- Need to add documentation for the 2 settings
- Need to modify the example query shape to now show field name and field data type and mention that we take this into account for the query shape.

### Issues Resolved
Closes #8539 

### Version
_List the OpenSearch version to which this PR applies, e.g. 2.14, 2.12--2.14, or all._

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@deshsidd Is this PR ready for review? And is it for 2.18? Thanks!

**Comment by deshsidd** [comment]
@kolchfa-aws You are quick was just about to tag you! :) Yes this is ready for review and its for 2.18.

File: _observing-your-data/query-insights/grouping-top-n-queries.md [hunk 1]
```diff
@@ -45,6 +45,96 @@ bool
 
 When queries share the same query structure, they are grouped together, ensuring that all similar queries belong to the same group.
 
+## Configuring query structure
```
Comment (natebower) [inline _observing-your-data/query-insights/grouping-top-n-queries.md:48]:
```suggestion
## Configuring the query structure
```

File: _observing-your-data/query-insights/grouping-top-n-queries.md [hunk 2]
```diff
@@ -45,6 +45,96 @@ bool
...
+      "type": "text"
+    },
+    "field4": {
+      "type": "long"
+    }
+  }
+}
+```
+
+The following query on the preceding index:
```
Comment (natebower) [inline _observing-your-data/query-insights/grouping-top-n-queries.md:73]: This needs a little more context/clarity of phrasing to indicate that it's connected to the next sentence and not the preceding one.

**Review by natebower — COMMENTED** [review]
@kolchfa-aws One comment and one change. Can you tag me on the rewrite of line 73? Thanks!

File: _observing-your-data/query-insights/grouping-top-n-queries.md (hunk 2 above)
Comment (kolchfa-aws) [reply _observing-your-data/query-insights/grouping-top-n-queries.md]: @natebower Reworded - please check.

**Review by natebower — APPROVED** [review]
@kolchfa-aws LGTM. Thanks!

**Pull request merged**